androidapp/app/src/main/assets/weather.json

```diff
@@ -1,3 +1,94 @@
 {
-  "TODO" : "FILL THIS WITH TEST DATA ONCE WE KNOW WHAT BACKEND IS PASSING AND WHAT IT LOOKS LIKE"
+  "sensor": {
+    "temperature": 0.00, // degrees C, check w/brian
```

Comment:
We may want to remove the comments, technically the json spec doesn't allow comments, there's a different spec & filetype for json with comments.

Reply:
Good catch, I have a linted version but forgot to replace that here. TODO

Reply:
- [ ] Remove comments in JSON files